v1/src/main/java/com/google/cloud/teleport/spanner/ddl/InformationSchemaScanner.java

```diff
@@ -156,18 +162,32 @@ Statement databaseOptionsSQL() {
         return Statement.of(
             "SELECT t.option_name, t.option_type, t.option_value "
                 + " FROM information_schema.database_options AS t "
-                + " WHERE t.catalog_name = '' AND t.schema_name = ''");
+                + " WHERE t.schema_name = ''");
       case POSTGRESQL:
         return Statement.of(
             "SELECT t.option_name, t.option_type, t.option_value "
                 + " FROM information_schema.database_options AS t "
-                + " WHERE t.schema_name NOT IN "
-                + "('information_schema', 'spanner_sys', 'pg_catalog')");
+                + " WHERE t.schema_name = 'public'");
```

Comment:
I hope this is a correct and backward compatible change. By backward compatible, I mean for all the existing pg customer databases, all the database options are stored with schema_name='public' is it?

Reply:
yes, all options are showed under default schema.